### DISCUSSION TITLE
Question: Is there a function to create a text node? something like createTextNode() in js?

### DISCUSSION THREAD

**Discussion by imadhui** [rabbibotton/clog#220] [Q&A] [answered]
Is there a function to create a text node? something like **createTextNode()** in js?

**Comment by rabbibotton** [comment]
No, but easy enough to add. Can you give me an example of how you would use?

**Reply by imadhui** [reply]
I want to add text inside div, but I don't want to replace the entire contents of the div node. I don't want add a div as a child with text in it either. I only want add a text node as child to the parent div node. How do I do that?

**Reply by rabbibotton** [reply]
with what I just added: (place-text-inside-bottom-of obj "text to add")

**Reply by rabbibotton** [reply]
Look forward to seeing the cool things you make with CLOG :)

**Comment by rabbibotton** [comment]
BTW, if you just want to replace a text node / add a text node in its entirety the "text" property (setf (text obj) "abc") does that.

I am looking in to appending or prepending text as that is the only thing comes to mind offhand.

**Comment by rabbibotton** [comment, marked as answer]
Ok, I added  place-text-inside-bottom-of and  place-text-inside-top-of - that is append text node, prepend text node and 'text' will replace all text nodes with a new text node

**Reply by imadhui** [reply]
This works! Thank you for the quick response. Great work, by the way. I'm inspired.
